spec/factories/spree/vendor_image_factory.rb

```diff
@@ -0,0 +1,5 @@
+FactoryBot.define do
+  factory :vendor_image, class: Spree::VendorImage do
+    attachment { Rack::Test::UploadedFile.new("#{Rails.root}/../fixtures/thinking-cat.jpg", 'image/jpeg') }
```

Comment:
@damianlegawiec I am new to image testing for engine,
Is there any better way to specify this path?
I had to use `Rails.root` to get dummy project path, and go one step outside with: `../` to reach fixtures folder.

Else the file would not be found.

Thank you :)

Reply:
https://github.com/spree/spree/blob/fd0131aea17c3a25972cf5079288bec7bd6870bc/backend/spec/controllers/spree/admin/stores_controller_spec.rb#L7 probably sth like this :)